auth/system-test/auth.test.js

```diff
@@ -16,7 +16,7 @@
 
 const path = require('path');
 const assert = require('assert');
-const tools = require('@google-cloud/nodejs-repo-tools');
+const childProcess = require('child_process');
```

Comment:
If `execSync` is the only property we use, I'd prefer this: 

```
const execSync = require('child_process').execSync;
```

Reply:
Sounds good! I'll do that on tests that are entirely covered by execSync.